Marlin/src/HAL/HAL_STM32/HAL_timers_STM32.cpp

```diff
@@ -71,7 +71,7 @@ bool timers_initialised[NUM_HARDWARE_TIMERS] = {false};
 void HAL_timer_start(const uint8_t timer_num, const uint32_t frequency) {
 
   if (!timers_initialised[timer_num]) {
-    constexpr uint32_t step_prescaler = STEPPER_TIMER_PRESCALE - 1,
+    uint32_t step_prescaler = STEPPER_TIMER_PRESCALE - 1,
```

Comment:
Why remove `constexpr`?

Reply:
Because it's not constant anymore and would give following compiler error:

```
../src/HAL/HAL_STM32/HAL_timers_STM32.h:62:50: error: call to non-constexpr function 'uint32_t HAL_RCC_GetSysClockFreq()'
```

Reply:
Ah, ok. I see that `STEPPER_TIMER_PRESCALE` isn't being used within any loops or ISRs so this all seems to be safe.